packages/remix/test/fixtures/02-remix-basics-mjs/remix.config.mjs

```diff
@@ -3,6 +3,7 @@
  */
 export default {
   ignoredRouteFiles: ["**/.*"],
+  serverBuildTarget: "vercel",
```

Comment:
Admittedly I'm a bit out-of-the-loop, but Remix docs say `serverBuildTarget` has been deprecated. Is this line just testing that `config.serverBuildTarget = undefined;` scrubs the config value?

Reply:
It is deprecated, but still exists in projects out in the wild. This PR is basically for backwards compat.